### PR TITLE
Use css linter

### PR DESCRIPTION
**Pull request by tcitworld**
Uses PostCSS + StyleLint in order to have proper css files.
Rules are the [stylelint-config-standard](https://github.com/stylelint/stylelint-config-standard) ones with 4 spaces tabulations.

### CODE REVIEW

**Comment by georgehrke** [comment]
please review @jancborchardt @raghunayyar

**Comment by tcitworld** [comment]
Also, the asset pipeline feature could be enabled now, since https://github.com/tchwork/jsqueeze/issues/39 is fixed.

**Comment by georgehrke** [comment]
@tcitworld Already done :) 
https://github.com/owncloud/calendar/pull/617

File: css/calendarlist.css [hunk 1]
```diff
@@ -97,69 +106,89 @@
...
 }
+
 #app-navigation .app-navigation-list-item .utils {
 	height: 44px;
 	padding: 0 5px 0 0;
 }
+
 #app-navigation .app-navigation-list-item .utils .action span {
-	opacity: .3;
+	opacity: 0.3;
```
Comment (jancborchardt) [inline css/calendarlist.css]: We use shorthand here always – no need for the extra 0.

File: css/calendarlist.css [hunk 2]
```diff
@@ -97,69 +106,89 @@
...
+	opacity: 0.7 !important;
 }
+
 #app-navigation .app-navigation-list-item .utils .action:hover span {
 	opacity: 1;
 }
+
 #app-navigation .app-navigation-list-item .active {
 	background: transparent !important;
+
```
Comment (jancborchardt) [inline css/calendarlist.css]: This line break seems to be unnecessary.

File: css/calendarlist.css [hunk 3]
```diff
@@ -97,69 +106,89 @@
...
 	/* Core Override */
 }
+
 #app-navigation .app-navigation-list-item .active a {
 	background-color: transparent !important;
+
```
Comment (jancborchardt) [inline css/calendarlist.css]: Same here.

File: css/calendarlist.css [hunk 4]
```diff
@@ -97,69 +106,89 @@
...
 	/* Core Override */
 }
+
 #app-navigation .app-navigation-list-item .active:hover .calendarlist-icon {
 	background-color: transparent !important;
+
```
Comment (jancborchardt) [inline css/calendarlist.css]: and here

File: css/calendarlist.css [hunk 5]
```diff
@@ -247,30 +284,39 @@ ul.dropdown-menu li > a:hover {
...
-
 /* Fullcalendar modifications */
 
 .fc th,
 .fc .fc-axis,
 .fc-day-grid-event .fc-time,
 .fc-ltr .fc-basic-view .fc-day-number,
 .fc-time-grid-event .fc-time {
-	opacity: .8;
+	opacity: 0.8;
```
Comment (jancborchardt) [inline css/calendarlist.css]: Shorthand is ok as well, here and below too.

File: css/datepicker.css [hunk 6]
```diff
@@ -21,16 +21,14 @@
...
-
 #datepicker .datepicker_current,
 #app-navigation .togglebuttons,
 #app-navigation .togglebuttons .btn-group {
 	float: left;
 }
 
 #app-navigation .togglebuttons .button.active {
-	background-color: rgba(255, 255, 255, .95);
+	background-color: rgba(255, 255, 255, 0.95);
```
Comment (jancborchardt) [inline css/datepicker.css]: Same here and below with the shorthand value.

File: css/calendarlist.css [hunk 7]
```diff
@@ -55,22 +59,27 @@
...
 	box-sizing: border-box;
 }
+
 #app-navigation .new-entity,
 #app-navigation .new-entity .new-entity-title {
 	cursor: pointer;
 }
-#app-navigation .new-entity .new-entity-title:before {
+
+#app-navigation .new-entity .new-entity-title::before {
```
Comment (jancborchardt) [inline css/calendarlist.css:68]: Is this really correct?

**Comment by jancborchardt** [comment]
@tcitworld nice, very cool optimization work! Just see my comments about 3 things: dot-value shorthands, ::before, and line breaks before comments. :)

File: css/calendarlist.css (hunk 7 above)
Comment (tcitworld) [inline css/calendarlist.css:68]: It is CSS3 valid : https://developer.mozilla.org/fr/docs/Web/CSS/::before

Comment (jancborchardt) [inline css/calendarlist.css:68]: Ah right, was just confused

**Comment by tcitworld** [comment]
I'll adjust the rules in order to fix those.

**Comment by tcitworld** [comment]
Fixed and rebased.

**Comment by jancborchardt** [comment]
Awesome stuff! Can’t see anything I disagree with :+1:

**Comment by georgehrke** [comment]
great stuff!
please squash and we can merge afterwords :)

**Comment by tcitworld** [comment]
There you go :)

**Pull request merged**